server/start.go

```diff
@@ -262,7 +263,18 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 	if err != nil {
 		return err
 	}
-	tmNode, err := optinode.NewNode(opticonv.GetNodeConfig(cfg), key, proxy.NewLocalClientCreator(app), ctx.Logger)
+	genesis, err := genDocProvider()
+	if err != nil {
+		return err
+	}
+	tmNode, err := optinode.NewNode(
+		context.Background(),
```

Comment:
Are the contexts passed to the method include Go's native context? Is there context to be used from existing code and not from the Background?

Reply:
Unfortunately no. I couldn't find any `context.Context` passed into this function, that's why I used `Background`.